georeference/georeference_shacl.ttl

```diff
@@ -0,0 +1,141 @@
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+georeference:GeoreferenceShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:ProjectLocationShape ;
+            sh:order 0 ;
+            sh:path georeference:projectLocation ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:GeodeticReferenceSystemShape ;
+            sh:order 1 ;
+            sh:path georeference:geodeticReferenceSystem ] ;
+    sh:targetClass georeference:GeoreferenceShape .
+
+georeference:GeodeticReferenceSystemShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234; Lon 45.23" ;
+            sh:description "World coordinates of map origin"@en ;
+            sh:message "Validation of origin failed!"@en ;
+            sh:name "origin"@en ;
+            sh:node georeference:Coordinate2DShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 0 ;
+            sh:path georeference:origin ],
+        [ skos:example "EPSG::32632" ;
+            sh:datatype xsd:string ;
+            sh:description "EPSG code of the map"@en ;
```

Comment:
can we add a sh:in here to have a list of valid values?

Reply:
No, there are thousands!

Reply:
fine with that